hypha/apply/projects/models/project.py

```diff
@@ -516,6 +516,26 @@ def __str__(self):
         return str(self.label)
 
 
+class ProjectReminderFrequency(Orderable, ClusterableModel):
+    num_days = models.IntegerField()
```

Comment:
Can we rename this to something like "reminder_days"? My browser thinks this is a 2FA field.